src/assets/translations/en/main.json

```diff
@@ -2211,7 +2211,8 @@
       "yourLoans": "Your Loans",
       "emptyTitle": "The Land of Zero Loans!",
       "emptyBody": "It appears you don't have any loans at the moment. Is this financial zen or just a break before your next big lending adventure? Either way, enjoy the calm!"
-    }
+    },
+    "unsafeDeposit": "This deposit has high slippage. Proceed with caution."
```

Comment:
Do we want to use borrow terminology vs. deposit? e.g
https://github.com/shapeshift/web/blob/b77d801603ab55cc541c11898a7c426044a506ee/src/assets/translations/en/main.json#L2199
https://github.com/shapeshift/web/blob/b77d801603ab55cc541c11898a7c426044a506ee/src/assets/translations/en/main.json#L2201

Reply:
For sure, consistency 🤌 

https://github.com/shapeshift/web/pull/6569/commits/6b2888c1d04597f110cd4fddbed0ee38867bc2c8